config/manifests/bases/odf-operator.clusterserviceversion.yaml

```diff
@@ -3,8 +3,13 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     alm-examples: '[]'
-    capabilities: Basic Install
+    capabilities: Deep Insights
```

Comment:
...do we actually qualify for Deep insights? I'm pretty sure ocs-operator doesn't, and I don't remember seeing that odf-operator exposes enough functionality for it either.

Reply:
I copied it from the ocs-operator itself

Reply:
Huh... okay! We'll see if anyone brings it up again. :laughing: